### PR TITLE
Add include parameter to project controller

### PR DESCRIPTION
**Pull request by jsokolowskii**
https://github.com/Activiti/Activiti/issues/4035

In order to extend the functionality of retrieving projects it is needed to add a new request param (`include={included-value-name}`) to already existing endpoints in ProjectController, e.g.
`/v1/projects?include={included-value-name}`
`/v1/projects/{project-id}?include={included-value-name}`

### CODE REVIEW

**Review by jesty — COMMENTED** [review]
What is the issue? Thanks

**Comment by jsokolowskii** [comment]
> What is the issue? Thanks

@jesty it's AAE-10074 and AAE-10075.

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/service/api/ProjectService.java [hunk 1]
```diff
@@ -59,11 +59,11 @@ Project copyProject(Project projectToCopy,
 
     Project replaceProjectContentWithProvidedModelsInFile(Project project, InputStream inputStream) throws IOException;
 
-    default Page<Project> getProjects(Pageable pageable, String name, List<String> filters) {
+    default Page<Project> getProjects(Pageable pageable, String name, List<String> filters, List<String> include) {
```
Comment (jesty) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/service/api/ProjectService.java]: why we are passing the value and then we are not using it?

Comment (jsokolowskii) [reply activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/service/api/ProjectService.java]: I will use it in enterprise repo where we implement this service. This way I won't need to change the implementation  in activiti-cloud.

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/service/decorators/ProjectDecorator.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/*
```
Comment (jsokolowskii) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/service/decorators/ProjectDecorator.java:1]: The code related to decorators and filters adds the possibility to implement those capabilities for getProject and getProjects endpoints.

**Review by jesty — APPROVED** [review]
LGTM

**Pull request merged**